api/migrations/dropWorkspaceSchema.sql

```diff
@@ -42,18 +28,19 @@ BEGIN
   RAISE NOTICE 'Schema ''%'' dropped.', v_schema;
 
   -- DROP v_role_readwrite as it inherit from v_role_readonly
-  EXECUTE format('DROP ROLE %I', v_role_readwrite);
+  EXECUTE format('DROP ROLE IF EXISTS %I', v_role_readwrite);
   RAISE NOTICE 'Role ''%'' dropped.', v_role_readwrite;
 
   -- REVOKE access to core.user
   EXECUTE format('REVOKE ALL ON core.lck_user FROM %I', v_role_readonly);
   EXECUTE format('REVOKE USAGE ON SCHEMA core FROM %I', v_role_readonly);
 
-  EXECUTE format('DROP ROLE %I', v_role_readonly);
+  EXECUTE format('DROP ROLE IF EXISTS %I', v_role_readonly);
```

Comment:
Is this still in WIP?

Reply:
Yes, I encounter some difficulties for dropping workspaces during tests.
As there is no deployment actually, I've thought this was better.
And, for those creation / dropping functions, maybe the code will be updated directly.
A migration could be added to update the function as they can be DROPed & CREATEd.